server/src/models/card.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (sequelize, DataTypes) => {
+  const Card = sequelize.define('Card', {
+    description: DataTypes.STRING,
+  }, {})
```

Comment:
@nikitasakov will `id` be acceptable for a Card class?

I mean can I call id of an object or just description?

Reply:
@AleksSenkou 
Strange thing :) Look.
First of all I don't know how to test our models like in railsc, because of that I used a crutch: send after creating `.then((card) => { res.status(201).send(card.description) })` and look what it return. 

So, if we send `card.description` we get:
<img width="928" alt="2018-12-03 11 02 03" src="https://user-images.githubusercontent.com/32716983/49360670-e9cb6000-f6ea-11e8-8f94-ab29f33bd0df.png">

But if we send `card.id` we get an empty json:
<img width="928" alt="2018-12-03 11 03 19" src="https://user-images.githubusercontent.com/32716983/49360725-12ebf080-f6eb-11e8-8c92-835b8d6c127d.png">

If we send `card.createdAt` everything is right again:
<img width="928" alt="2018-12-03 11 04 34" src="https://user-images.githubusercontent.com/32716983/49360805-4a5a9d00-f6eb-11e8-9612-847941b04399.png">


And now strange thing: if we send `card.id + card.description` we get and `id` and `description`. I can't understand why it can't return alone id :)
<img width="928" alt="2018-12-03 11 07 42" src="https://user-images.githubusercontent.com/32716983/49360937-acb39d80-f6eb-11e8-86bf-4657ebab3f0c.png">

Reply:
@nikitasakov which tool have you used? I'd like to try

Reply:
@AleksSenkou Postman

Reply:
@nikitasakov can you give an example of a request as a postman screenshot?

Reply:
@AleksSenkou 
<img width="1440" alt="2018-12-03 18 32 30" src="https://user-images.githubusercontent.com/32716983/49383474-e3100d80-f729-11e8-8ca6-baf585643da4.png">

Reply:
@AleksSenkou Yes, I got this response too, try to write `.then((card) => { res.status(201).send(card.id) })`

and others variations (`card.createdAt` etc)

Reply:
When sending just `id` it says:

`express deprecated res.send(status): Use res.sendStatus(status) instead src/controllers/v1/cards.js:13:23`

That means that express doesn't understand that you want to send an id because response should be an object, not just value

That's why:

```js
.then((card) => { res.status(201).send({id: card.id}) })
```

will return object with `id` value

Reply:
@datarockets/cuties ^ please, pay more attention on what logs say 😉

Reply:
@AleksSenkou But what is the difference between `id` and other parameters? As I understand `description` is not object too, but we can send `card.description`.

Reply:
@nikitasakov because `id` is numeric. And `res.send(number)` previously was used for sending http codes

at the moment `send(number)` is deprecated and it raises a warning and returns `{}`

Reply:
Cool, conversation can be resolved!